.github/workflows/release.yaml

```diff
@@ -52,3 +57,27 @@ jobs:
             -H "Accept: application/vnd.github.v3+json" \
             https://api.github.com/repos/${{github.repository}}/actions/workflows/${{env.QUILL_ICONS_PARK_WORK_FLOW_FILE}}/dispatches \
             -d '{"ref":"main"}'
+
+  send_slack_notification:
+    name: Send Release Slack notification
+    runs-on: ubuntu-latest
+    if: always()
+    needs: [release]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```

Comment:
`actions/checkout@v4` ?

Reply:
Hi @shaheer-deriv, this is the way that security team recommended to deal with third party actions. I'll DM you the  guide.

Reply:
Thanks @jim-deriv 🙏